enhancements/rhcos/liveisoinstall.md

```diff
@@ -0,0 +1,93 @@
+---
+title: RHEL CoreOS Live ISO/installer
+authors:
+  - "@cgwalters"
+reviewers:
+  - "@coreos-team"
+approvers:
+  - "@coreos-team"
+creation-date: 2020-02-07
+last-updated: 2020-02-07
+status: implementable
+---
+
+# Rebase RHEL CoreOS 4.5 on Fedora CoreOS $latest
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+RHEL CoreOS is built from RHEL components, but uses configuration and tooling from Fedora CoreOS.  They share many things such as the installer code, etc.
+
+This enhancement tracks rebasing RHCOS 4.5 on the latest FCOS code, in particular the installer.
+
+### New "Live" installer
```

Comment:
I feel like I'm missing something fundamental, how does the new live installer differ from current installer in non-interactive flows does the live installer handle ignition without the reboot or does it remain unchanged?

Reply:
It's not fundamentally different in features or flow (though it has been rewritten in Rust instead of shell script).

Reply:
In the new model there are two distinct Ignition configs: the one applied to the installed system, just as now, and a second one applied _to the live system performing the installation_.  That second one is how the install process can be customized non-interactively.